File: TUGAS DPBO/Latihan 1/B/python/club.py
class club():

    __name = ""
    __country = ""
    __year = ""
    __player = ""
    __coach = ""

    def __init__(self, name = "", country = "", year = "", player = "", coach = ""):
        self.__name = name
        self.__country = country
        self.__year = year
        self.__player = player
        self.__coach = coach

    def setName(self, name):
        self.__name = name

    def setCountry(self, country):
        self.__country = country

    def setYear(self, year):
        self.__year = year

    def setPlayer(self, player):
        self.__player = player

    def setCoach(self, coach):
        self.__coach = coach

    def getName(self):
        return self.__name

    def getCountry(self):
        return self.__country

    def getYear(self):
        return self.__year

    def getPlayer(self):
        return self.__player

    def getCoach(self):
        return self.__coach 

File: TUGAS DPBO/Latihan 1/B/php/club.php
<?php

class Club{
    private $Name = "";
    private $Country = "";
    private $Year = "";
    private $Player = "";
    private $Coach = "";
    
    public function __construct($Name = "", $Country = "", $Year = "", $Player = "", $Coach = ""){
        $this->Name = $Name;
        $this->Country = $Country;
        $this->Year = $Year;
        $this->Player = $Player;
        $this->Coach = $Coach;
    }

    public function setName($Name){
        $this->Name = $Name;
    }

    public function getName(){
        return $this->Name;
    }

    public function setCountry($Country){
        $this->Country = $Country;
    }

    public function getCountry(){
        return $this->Country;
    }

    public function setYear($Year){
        $this->Year = $Year;
    }

    public function getYear(){
        return $this->Year;
    }

    public function setPlayer($Player){
        $this->Player = $Player;
    }

    public function getPlayer(){
        return $this->Player;
    }

    public function setCoach($Coach){
        $this->Coach = $Coach;
    }

    public function getCoach(){
        return $this->Coach;
    }
}  
?>
